runtime/oti/j9nonbuilder.h

```diff
@@ -5653,6 +5653,12 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t jclCacheMutex;
 	UDATA arrayletLeafSize;
 	UDATA arrayletLeafLogSize;
+	UDATA sizeOfJ9IndexableObjectContiguous;
+	UDATA sizeOfJ9IndexableObjectDiscontiguous;
+	BOOLEAN isDataAddrPresent;
```

Comment:
this should probably be under J9VM_ENV_DATA64, too

Reply:
I believe `isDataAddrPresent` could/should be, but shouldn't the other 2 should remain outside of `J9VM_ENV_DATA64`?

Reply:
right, i meant the flag only